webapp/src/actions.js

```diff
@@ -129,9 +131,19 @@ export const bump = (id) => async (dispatch, getState) => {
 };
 
 export function autocompleteUsers(username) {
-    return async (doDispatch) => {
-        const {data} = await doDispatch(UserActions.autocompleteUsers(username));
-        return data.users;
+    return async (doDispatch, getState) => {
+        let userIds = [];
+        let users = [];
+        let team = TeamSelector.getCurrentTeam(getState())
+        const teamMembers = await doDispatch(TeamActions.getTeamMembers(team.id));
+        for( let j =0; j< teamMembers.data.length; j++) {
+            const user = await doDispatch(UserActions.getUser(teamMembers.data[j].user_id));
```

Comment:
It seems we've changed the expected functionality, and made this a bit unperformant.

Originally:

- We used the `autocompleteUsers` to search for a user by username

Now:

- We fetch all team members of the current team from the API
- Then individually fetch each user of that team from the API

It looks like the `autocompleteUsers` function accepts a `teamId` argument:

https://github.com/mattermost/mattermost-webapp/blob/b506465285bd4e95d2242349046ebb9cc95e6385/packages/mattermost-redux/src/actions/users.ts#L908

Can we use that existing function instead of introducing the implementation used below?